packages/api2teams/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/api2teams",
-  "version": "0.0.1",
+  "version": "0.1.0",
```

Comment:
do we need to use version 0.1.0-rc

Reply:
rc should be added in npm version step in cd @wenytang-ms

Reply:
@SLdragon As sync with @wenytang-ms , we need to add -rc to publish 0.1.0-rc.0